pprime/pprime.py

```diff
@@ -0,0 +1,54 @@
+"""
+ID: whuan2001
+LANG: PYTHON2
+TASK: pprime
+"""
+import math
+import cProfile
+def solution():
+    fin = open("pprime.in", "r")
+    fout = open("pprime.out", "w")
+    a, b = map(int, fin.readline().strip().split())
+    res = []
+    if a < 10:
+        for k in range(a, 10):
+            if prime(k):
+                res.append(k)
+    for num in range(1, 10000):
+        palindromes = creat_palindrome(num, a, b)
+        for palindrome in palindromes:
+            if palindrome % 2 != 0 and prime(palindrome):
+                res.append(palindrome)
+    res = sorted(res)
+    for r in res:
+        print >>fout, r
+def creat_palindrome(num, a, b):
+    digits = []
```

Comment:
I would recommend to use int(str(num) + str(num)[::-1]) to replace while loop here

str(1234) => "1234"
str(1234)[::-1] => "4321"
int("43211234") => 43211234